### PR TITLE
Add: TabRegex

### PR DESCRIPTION
**Pull request by arnav-kr**
### Project Details
1. **What is the name of your project?** 
    TabRegex

3. **How many hours did you spend developing this?**  
   9 hrs

4. **What problem does your extension solve, or what makes it unique?**  
  The Extension solves the problem of multiple tabs, which for a normal person almost keep getting accumulated over time. 
  It provides a way to simply just enter a regex, and close all the tabs that matches that specific regex

5. **How many people have tested your extension?**  
   4

6. **Does your extension work in both Chrome and Firefox?**  
  Yes

---

### Additional Comments
GitHub Repo: https://github.com/arnav-kr/TabRegex

### CODE REVIEW

File: submission/TabRegex/.github/FUNDING.yml
Comment (JosiasAurel) [inline submission/TabRegex/.github/FUNDING.yml:1]: please remove this

File: submission/TabRegex/.github/ISSUE_TEMPLATE/bug_report.yaml
Comment (JosiasAurel) [inline submission/TabRegex/.github/ISSUE_TEMPLATE/bug_report.yaml:1]: please remove this

File: submission/TabRegex/.github/ISSUE_TEMPLATE/feature_request.yaml
Comment (JosiasAurel) [inline submission/TabRegex/.github/ISSUE_TEMPLATE/feature_request.yaml:1]: remove this too

**Review by JosiasAurel — CHANGES_REQUESTED** [review]
please remove the files i've marked. that said, i love the idea

**Comment by arnav-kr** [comment]
oh, actually i directly added the git subtree from my repo thats why they're there, I'll remove those

**Pull request merged**